CHANGELOG

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Dec  7 08:31:05 UTC 2016 - jreidinger@suse.com
+
+- Use correct configuration file for xen kernel parameters
```

Comment:
@jreidinger AFAICT this description is wrong. We have not changed *which file* is being written. We have changed the *contents* of the file, namely by *using the correct configuration key*.
To be nice to the reader, we could explicitly say what keys are involved, and what is the user visible impact of the bug/fix:
> Use GRUB_CMDLINE_XEN_DEFAULT instead of GRUB_CMDLINE_LINUX_XEN to fix kdump enabling (bsc...)